knowledge-base/kb-filedialogsdemos-notavailable.md

```diff
@@ -0,0 +1,37 @@
+---
+title: FileDialogs Demos Not Available in WPF Control Examples App
```

Comment:
This should be more generic, for all controls in the application, not only the FileDialogs.
Reflect this change to all places in the article where we talk about the FileDialogs explicitly.